vizro-core/src/vizro/static/css/table.css

```diff
@@ -1,10 +1,19 @@
+#right-side .dash-cell {
+  height: 48px !important;
```

Comment:
Did this hurt 😂 ?

Reply:
Yes 😅  I actually wished that we wouldn't have the DataTable in our code basis anymore